packages/app/src/app/pages/Dashboard/Content/routes/Settings/UserSettings/WorkspaceSettings.tsx

```diff
@@ -200,11 +197,6 @@ export const WorkspaceSettings = () => {
                     onClick={() => setEditing(false)}
                   />
                 </Stack>
-                {isFree && (
```

Comment:
Yes, this is the component for the Personal space settings, and there's no more free badge for that, as it will always be free, with the removal of Personal Pro